x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -212,6 +227,28 @@ const transformNewSubCase = ({
   };
 };
 
+/**
+ * This type should only be used within the cases service and its helper functions (e.g. the transforms).
+ *
+ * The type represents how the external services portion of the object will be layed out when stored in ES. The external_service will have its
+ * connector_id field removed and placed within the references field.
+ */
+export type ExternalServicesWithoutConnectorId = Omit<
```

Comment:
Given that this type is used by the transforms I think is best to create a `type.ts` file to avoid circular dependencies.